modules/libmicroros/microros_transports/udp/microros_transports.h

```diff
@@ -18,8 +18,13 @@
 #include <unistd.h>
 
```

Comment:
```suggestion
#include <version.h>
```

We need to include the `version.h` file to use ZEPHYR_VERSION after